pages/dkp/kaptain/2.0.0/sdk/1.0.x/release-notes/index.md

```diff
@@ -6,29 +6,19 @@ beta: false
 menuWeight: 0
 ---
 
-Kaptain SDK 1.0.0 Release Notes
+Kaptain SDK 1.0.1 Release Notes
 
 ### New Features
-* Support for model building/deploying using non-deep learning frameworks
-* Added support for new frameworks: scikit-learn, XGBoost, MXNet, LightGBM and ONNX
-* Added two new tutorials: [Developing and deploying scikit-learn models with Kaptain SDK][quick-start] and [Build Docker Images with Kaptain SDK][image-builder]
 
-### Breaking changes
-`Model.framework` property type has been changed from `str` to `ModelFramework(Enum)` and need to be updated based on the actual model framework, e.g:
+No new features for this release.
```

Comment:
@juliangieseke added this just to have something underneath the header and indirectly provide the info that this release is only about improvement and fixes.

Reply:
I think Id rather remove the header entirely (missed that). Its more common not to have the header rather than stating no changes.